pkgs/top-level/config.nix

```diff
@@ -140,9 +182,26 @@ let
       feature = "build packages with ROCm support by default";
     };
 
+    packageOverrides = mkOption {
+      type = types.functionTo types.raw;
+      default = lib.id;
```

Comment:
This default is due to https://github.com/NixOS/nixpkgs/actions/runs/10137454574/job/28027709399?pr=330753